src/axom/inlet/inlet_utils.hpp

```diff
@@ -28,6 +28,33 @@ namespace inlet
 */
 void setWarningFlag(axom::sidre::Group* root);
 
+/*!
+*****************************************************************************
+* \brief This function is used to configure the target group as required

```

Comment:
This could be clarified by changing`target group` to something like `Inlet item(??) held at the given Sidre group, target, .."

Same with checkRequired